zlib/contrib/minizip/ioapi.h.patch

```diff
@@ -0,0 +1,11 @@
+--- ../zlib-1.2.8/contrib/minizip/ioapi.h	2012-01-16 19:51:31.000000000 -0700
++++ zlib/contrib/minizip/ioapi.h	2017-11-11 09:05:50.662012999 -0700
```

Comment:
Where do these patches come from? I do not see them in https://github.com/madler/zlib/blob/21767c654d31d2dccdde4330529775c6c5fd5389/contrib/minizip/ioapi.h.